cypress/e2e/19-execution.cy.ts

```diff
@@ -214,6 +214,33 @@ describe('Execution', () => {
 		workflowPage.getters.clearExecutionDataButton().should('not.exist');
 	});
 
+	it('should test workflow with specific trigger node', () => {
+		cy.createFixtureWorkflow('Manual_webhook_wait.json');
+		cy.intercept('POST', '/rest/workflows/**/run?**').as('workflowRun');
+
+		workflowPage.getters.zoomToFitButton().click();
+		workflowPage.getters.executeWorkflowButton('Manual').should('not.be.visible');
+
+		// Hover the manual trigger node
+		workflowPage.getters.canvasNodeByName('Manual').realHover();
+		workflowPage.getters.executeWorkflowButton('Manual').should('be.visible');
+
+		// Execute the workflow
+		workflowPage.getters.executeWorkflowButton('Manual').click();
+
+		// Check workflow buttons
+		workflowPage.getters.executeWorkflowButton().get('.n8n-spinner').should('be.visible');
+		workflowPage.getters.executeWorkflowButton('Manual').should('be.disabled');
+
+		successToast().contains('Workflow executed successfully');
+
+		cy.wait('@workflowRun').then((interception) => {
```

Comment:
Why are we checking the API request? Might be more maintainable to check the output of the node here, that the nodes have turned green or that the NDV shows output data, instead of coupling it to the API. 

I recommend the schedule node since it always outputs the same data.